src/assets/i18n/en.json5

```diff
@@ -4635,7 +4635,7 @@
 
   "submission.workflow.generic.delete": "Delete",
 
-  "submission.workflow.generic.delete-help": "If you would to discard this item, select \"Delete\".  You will then be asked to confirm it.",
+  "submission.workflow.generic.delete-help": "Select this option if you would to discard this item, select \"Delete\".  You will then be asked to confirm it.",
```

Comment:
This change is in the correct direction, because it keeps the help/hint similar to the other buttons.  But, this new sentence doesn't make sense in English.  So, I'd recommend updating it to say this:
`Select this option to discard this item. You will then be asked to confirm it.`

Reply:
That is true @tdonohue sorry for my mistake, i will do the change inmediatly

Reply:
@tdonohue The modification you suggested is already done 👍